claimer/go.mod

```
@@ -1,19 +1,23 @@
 module github.com/kava-labs/go-tools/claimer
 
-go 1.14
+go 1.16
 
 require (
-	github.com/cosmos/cosmos-sdk v0.39.2
-	github.com/gorilla/mux v1.7.4
+	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/gorilla/mux v1.8.0
 	github.com/kava-labs/binance-chain-go-sdk v1.2.5-kava
-	github.com/kava-labs/go-sdk v0.3.0-rc1
-	github.com/kava-labs/go-tools/deputy-claimer v0.0.0-20201223131958-31f436616e96
-	github.com/kava-labs/kava v0.12.1
-	github.com/sirupsen/logrus v1.4.2
+	github.com/kava-labs/go-sdk v0.5.1-0.20211220154055-4aeeffe85ecd
+	github.com/kava-labs/go-tools v0.0.0-20220105190757-b0a6146a2540
+	github.com/kava-labs/kava v0.16.0-rc1.0.20220111173147-4615cef9393b
+	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.6.1
-	github.com/tendermint/tendermint v0.33.9
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
+	github.com/stretchr/testify v1.7.0
+	github.com/tendermint/tendermint v0.34.14
+	golang.org/x/sys v0.0.0-20211210111614-af8b64212486 // indirect
+	google.golang.org/grpc v1.43.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
 
 replace github.com/kava-labs/go-tools/deputy-claimer => ../deputy-claimer
+
+replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
```

```suggestion
replace (
	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
	google.golang.org/grpc => google.golang.org/grpc v1.33.2
)
```

Not sure if any queries in claimer would have the problem but just to be safe:

https://docs.cosmos.network/master/core/grpc_rest.html#grpc-server

Also, maybe we could do the same as auction-bot and deputy-claimer and just remove the go.mod/.sum files, move needed deps to the root go.mod and not need to import the go-tools package to use shared code.  Not sure if we want to bring extra unneeded deps into other packages though like mux -- doesn't seem like there's a great way to do monorepos with go